packages/contracts-bedrock/contracts/L2/BVM_BIT.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+/* Library Imports */
+import { Predeploys } from "../libraries/Predeploys.sol";
+
+/* Contract Imports */
+import { MantleMintableERC20 } from "../universal/MantleMintableERC20.sol";
+
+/**
+ * @title BVM_BIT
+ * @dev The ETH predeploy provides an ERC20 interface for ETH deposited to Layer 2. Note that
+ * unlike on Layer 1, Layer 2 accounts do not have a balance field.
+ */
+contract BVM_BIT is MantleMintableERC20 {
+    /***************
+     * Constructor *
+     ***************/
+    // hardcode to bit token mainnet address
+    constructor()
+        MantleMintableERC20(Predeploys.L2_STANDARD_BRIDGE,address(0x1A4b46696b2bB4794Eb3D4c26f1c55F9170fa4C5),"Bit Token","BIT")
```

Comment:
Can a constant be used to define this address?

Reply:
Where is the private key of `0x1A4b46696b2bB4794Eb3D4c26f1c55F9170fa4C5`?